src/ZIndexContext.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+
+const ZIndexContext = React.createContext(0)
```

Comment:
Wondering if we should rename this so it matches the problem domain instead of the technical domain, e.g. `LayerContext` and exporting: `LayerProvider` and `LayerConsumer`.

The prop can still be called `zIndex` in **MenuWrapper**.

Reply:
Yes, love it.